machine-learning/tutorials/SentimentAnalysis/Program.cs

```diff
@@ -116,15 +116,14 @@ public static void Evaluate(MLContext mlContext, ITransformer model, IDataView s
             CalibratedBinaryClassificationMetrics metrics = mlContext.BinaryClassification.Evaluate(predictions, "Label");
             // </SnippetEvaluate>
 
-            // The Accuracy metric gets the accuracy of a classifier, which is the proportion 
+            // The Accuracy metric gets the accuracy of a model, which is the proportion 
             // of correct predictions in the test set.
 
-            // The Auc metric gets the area under the ROC curve.
-            // The area under the ROC curve is equal to the probability that the classifier ranks
+            // The AreaUnderROCCurve metric is equal to the probability that the algorithm ranks
             // a randomly chosen positive instance higher than a randomly chosen negative one
             // (assuming 'positive' ranks higher than 'negative').
 
-            // The F1Score metric gets the classifier's F1 score.
+            // The F1Score metric gets the model's F1 score.
             // The F1 score is the harmonic mean of precision and recall:
             //  2 * precision * recall / (precision + recall).
```

Comment:
would it be simpler to just say: F1 is a measure of tradeoff between precision and recall.